SIL.ReleaseTasks/SetReleaseNotesProperty.cs

```diff
@@ -190,12 +149,39 @@ private string ConvertLatestChangelog(int level, int skipUntilLevel = -1)
 							_currentIndex = _markdownLines.Length;
 						break;
 					default: 
-						if (currentLine.StartsWith("-") && _filterRegex.IsMatch(currentLine) && FilterEntries)
+						if (FilterEntries)
 						{
-							if (currentLine.StartsWith($"- [{PackageId}]"))
+							int m = _currentIndex;
+							if (_filterRegex.IsMatch(currentLine) && !String.IsNullOrEmpty(currentLine))
+							{
+								if (currentLine.StartsWith($"- [{PackageId}]"))
+								{
+									string newLine = currentLine.Replace($" [{PackageId}]", "");
+									bldr.AppendLine(newLine);
+									m++;
+									while (m < _markdownLines.Length && !_markdownLines[m].StartsWith("- [") && !String.IsNullOrEmpty(_markdownLines[m]))
+									{
+										bldr.AppendLine(_markdownLines[m]);
+										_currentIndex = m;
+										m++;
+									}
+									
+								}else
+								{
+									while (m < _markdownLines.Length && !_markdownLines[m].StartsWith("- [") && String.IsNullOrEmpty(_markdownLines[m]))
+									{
+										_currentIndex = m;
+										m++;
+									}
+								}
+							}else if (currentLine.StartsWith("-") && !String.IsNullOrEmpty(currentLine))
```

Comment:
```suggestion
							}
							else if (currentLine.StartsWith("-"))
```

We already checked for an empty line above the switch statement.